math/wide_integer/uintwide_t.h

```diff
@@ -316,21 +316,21 @@
     }
 
   protected:
-    mutable size_type elem_count;
-    pointer           elems;
+    mutable size_type elem_count; //NOLINT(cppcoreguidelines-non-private-member-variables-in-classes,misc-non-private-member-variables-in-classes)
```

Comment:
Any reason not to make this `private`? It's quite rare (if ever) that `protected` is the best choice.

Reply:
I need to look more deeply into this. Also the general discussion of protected versus never-protected is of great interest to me as i evolve in C++ design practices.

Reply:
Done. Taken from @johnmcfarlane #164